packages/storybook/stories/segmented-control.mdx

```diff
@@ -0,0 +1,41 @@
+import { ToggleButtonGroup } from '@equisoft/design-elements-react';
+import { ArgTypes, Meta, Canvas } from '@storybook/blocks';
+import * as ToggleButtonGroupStories  from './segmented-control.stories';
+
+<Meta of={ToggleButtonGroupStories} />
+
+# Segmented Control
+1. [Definition](#definition)
+2. [Usage](#usage)
+3. [Variants](#variants)
+4. [Properties](#properties)
+
+## Definition
+A segmented control is a horizontal set of 2 to 5 exclusive segments, each functioning as a mutually exclusive button. Segmented controls provide closely related choices that affect an object, state, or view [2][3][4]. 
```

Comment:
Est-ce qu'on pourrait simplifier?
"_Segmented control is a row of 2 to 5 exclusive buttons where only one can be selected at a time. They provide closely related choices to control an object, state, or view._"

Reply:
Aurais-tu un example où ca viendrait contrôler le "_state_" d'un object?

Reply:
Exemples: 

Object: L'alignement à gauche, centré ou à droite d'un texte.
State: Liste ordonnée selon les nouvelles publications, les plus vieilles ou les plus populaires. 
View: Vue d'un projet en table, graphique ou timeline. 

Source: https://primer.style/components/segmented-control

Reply:
Est-ce que l'exemple du "_State_" est en quelque sorte une façon de filtrer?
Dans la première version de la documentation, tu indiquais de ne pas utiliser le composant pour le filtrage et de plutôt opter pour les Tabs. Tu as retiré cette mention après que j'ai posé la question sur l'utilisation des Tabs pour le filtrage, mais la question demeure: tu mentionnais quand même de ne pas utiliser le composant à des fins de filtrage.

Reply:
J'ai retiré la phrase qui mentionnait de ne pas filtrer. Je me suis fiée à Material UI et ils ont l'option de l'utiliser comme ceci:

« **Filter control:** Activating a segment affects the visibility of items in a list, and there is a segment that shows the full list. »

Je trouve que c'est plus clair ainsi, donc se fier à cette version.